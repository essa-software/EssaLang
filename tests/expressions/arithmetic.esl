// output: 15,5,50,2,0

func main() {
    let a = 10;
    let b = 5;
    print("{},", a + b);
    print("{},", a - b);
    print("{},", a * b);
    print("{},", a / b);
    print("{}", a % b);
    a + 5542;
}
