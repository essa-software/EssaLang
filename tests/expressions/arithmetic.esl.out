15,5,50,2,0